lib/auth/auth.go

```diff
@@ -1256,6 +1257,12 @@ func (a *Server) runPeriodicOperations() {
 	})
 	defer dynamicLabelsCheck.Stop()
 
+	notificationsCleanup := interval.New(interval.Config{
+		Duration:      2 * 24 * time.Hour, // every 2 days
+		FirstDuration: utils.HalfJitter(time.Second * 10),
+		Jitter:        retryutils.NewSeventhJitter(),
+	}).Next()
```

Comment:
```suggestion
	})
```
I'd drop the `.Next()` here and put that into the select below.